.github/iac/main.tf

```diff
@@ -76,3 +82,17 @@ module "fabric_rpc" {
   email_ses_verified_email    = local.email_ses_verified_email
   url_prefix                  = "rpc"
 }
+
+module "fabric_daemon" {
+  source = "../../bootstrap/daemon"
+
+  namespace      = local.daemon_namespace
+  image          = var.daemon_image
+  cluster_id     = ""
```

Comment:
This should be an identifier for the m2 cluster. I would put `txpipe-us-east-2-m2`

Reply:
cool! I will update it